tools/run_session

```diff
@@ -199,7 +198,7 @@ mission_cleanup() {
     echo -e '\n'
 }
 
-
+world_dir
```

Comment:
What does this line do? If it's not necessary, we should remove it.

Reply:
Fixed. This looks like one of the search terms I had been using :). Sorry about that.